tests/test_mock_twilio.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+import json
+
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from app.models import db, PhoneNumberVerification
+
+db.create_all()
+
+input = json.loads('''{
+  "caller_name": {
+    "caller_name": "Delicious Cheese Cake",
+    "caller_type": "CONSUMER",
+    "error_code": null
+  },
+  "carrier": null,
+  "country_code": "US",
+  "national_format": "(510) 867-5310",
+  "phone_number": "+15108675310",
+  "add_ons": null,
+  "url": "https://lookups.twilio.com/v1/PhoneNumbers/+15108675310"
+}''')
```

Comment:
can you try putting this in a separate file? we can add a directory `tests/fixtures/` and put this into `tests/fixtures/phone_number_with_caller_name.json`

Then, we can add a file `tests/fixtures/phone_number_missing_caller_name.json` for the test where `input['caller_name']` is `null`.